### ISSUE TITLE
License choice

### ISSUE THREAD

**Issue by katerinkus** [UBC-MDS/collinearityR_tool#31]
Let's discuss what would be a good license option for this R package.

**Comment by katerinkus** [comment]
I suggest Mozilla Public License 2.0 license ([details](https://choosealicense.com/licenses/mpl-2.0/)). In our case, the code is not innovative so people already probably write functions like these for themselves. If someone wants to reuse chunks of our code, I do not see it as any meaningful loss to us. In addition, this license requires that people who use our code, also have to put their code up under the **same** license. As a result, this work keeps propagating and improving further without getting behind closed doors. What do you think team?

**Comment by showcy** [comment]
> I suggest Mozilla Public License 2.0 license ([details](https://choosealicense.com/licenses/mpl-2.0/)). In our case, the code is not innovative so people already probably write functions like these for themselves. If someone wants to reuse chunks of our code, I do not see it as any meaningful loss to us. In addition, this license requires that people who use our code, also have to put their code up under the **same** license. As a result, this work keeps propagating and improving further without getting behind closed doors. What do you think team?

Sounds good to me.

**Comment by nickmao1994** [comment]
I agree with using a weak license since our work is not that innovative. I think we are open to anyone who wants to use it.
I would suggest MIT or Mozilla Public License 2.0. They both look suitable in our case. Comparing with Mozilla, MIT allows licensed works, modifications, and larger works to be distributed under different terms and without source code. This might give more flexibility to others who are using our work.

**Comment by showcy** [comment]
>I agree with using a weak license since our work is not that innovative. I think we are open to anyone who wants to use it.
I would suggest MIT or Mozilla Public License 2.0. They both look suitable in our case. Comparing with Mozilla, MIT allows licensed works, modifications, and larger works to be distributed under different terms and without source code. This might give more flexibility to others who are using our work.

I also prefer to MIT. With MIT, people have to mention their work is built based on our package but they do not need to explicitly explain how much our package contributed to their work or the way how they use our package. 

As ours is MIT now, is it good to keep it as it?

**Comment by Anahita97** [comment]
I also agree that MIT seems to be suitable for our work. As you guys have mentioned, the MIT license is friendly to both open source developers and businesses. Moreover, the license seems to be pretty short in comparison to some other open-source licenses, and therefore the user can quickly learn about the license terms.
Therefore, due to the aforementioned reasons, I agree that the MIT license is suitable for our work!
